README.md

```diff
@@ -20,6 +20,7 @@ Sessions that are exceptionally mentionable are highlighted using a ★.
 Feel free to submit a `PR` if I got something wrong or you have a suggestion for improvement. Please also have a look in [CONTRIBUTING.md](CONTRIBUTING.md) if you want to contribute.
 
 Thanks so much to EVERYBODY who contributed and improved the overall quality of the notes and those who added complete notes to the list:
+[@mcichecki](https://github.com/mcichecki)
```

Comment:
This year i don't maintain the contributors list since the new Github Design shows them nicely at the right sidebar (you can enable this by tapping on your avatar and select `feature previews`). The colon behind `list` was planned to be a `.`. So pleae take it easy when I remove this line after the PR is merged.